tools/cli/src/commands/deploy.rs

```diff
@@ -0,0 +1,200 @@
+use crate::context::AppContext;
+use anyhow::{anyhow, bail, Result};
+use deadpool::managed::Pool;
+use lavs_task_queue::msg::{Requestor, TimeoutInfo};
+use layer_climb::prelude::*;
+use std::path::PathBuf;
+use tokio::try_join;
+
+pub async fn deploy_contracts(ctx: AppContext, artifacts_path: PathBuf) -> Result<()> {
+    tracing::info!("Uploading contracts from {:?}", artifacts_path);
+
+    let wasm_files = WasmFiles::read(artifacts_path.clone()).await?;
+
+    let CodeIds {
+        operators: operators_code_id,
+        task_queue: task_queue_code_id,
+        verifier_simple: verifier_code_id,
+    } = CodeIds::upload(wasm_files, ctx.client_pool.clone()).await?;
+
+    tracing::info!("Contracts all uploaded successfully, instantiating...");
+
+    let client = ctx.client_pool.get().await.map_err(|e| anyhow!("{e:?}"))?;
+
+    let (operators_addr, tx_resp) = client
+        .contract_instantiate(
+            client.addr.clone(),
+            operators_code_id,
+            "Mock Operators",
+            &lavs_mock_operators::msg::InstantiateMsg {
+                operators: vec![lavs_mock_operators::msg::InstantiateOperator::new(
+                    client.addr.to_string(),
+                    1,
+                )],
+            },
+            vec![],
+            None,
+        )
+        .await?;
+
+    tracing::info!("Mock Operators Tx Hash: {}", tx_resp.txhash);
```

Comment:
A bit too much as `info` IMO.
Can the tx hashes go to `debug` level?
In fact, if they are all reported at the bottom, maybe all these intermediate updates can be `debug` (as only needed to debug when it goes wrong and normally not needed by users)

Reply:
done in https://github.com/Lay3rLabs/avs-toolkit/pull/28/commits/71f31dd5db101b48abdd3d3f02729030d95313f8